releasenotes/notes/kafka-support-serializing-producer-fb01f5288ae116f4.yaml

```diff
@@ -1,3 +1,3 @@
 features:
   - |
-    kafka: Add support for the Kafka serializing producer and deserializing consumer.
+    kafka: Adds support for the Kafka serializing producer and de serializing consumer.
```

Comment:
We can keep `deserializing` and just add it to the `spelling_wordlist.txt` file, like so: https://github.com/DataDog/dd-trace-py/blob/1.x/docs/spelling_wordlist.txt